plugins/scaffolder-backend-module-github/src/actions/githubRepoCreate.examples.ts

```diff
@@ -63,4 +63,626 @@ export const examples: TemplateExample[] = [
+      'Set repository visibility to internal with description and homepage.',
+    example: yaml.stringify({
+      steps: [
+        {
+          action: 'github:repo:create',
+          name: 'Create a new internal GitHub repository with description and homepage',
+          input: {
+            repoUrl: 'github.com?repo=repo&owner=owner',
+            repoVisibility: 'internal',
+            description: 'Internal repository for team collaboration',
+            homepage: 'https://internal.example.com',
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Disable auto-merge.',
+    example: yaml.stringify({
+      steps: [
+        {
+          action: 'github:repo:create',
+          name: 'Create a new GitHub repository with auto-merge disabled',
+          input: {
+            repoUrl: 'github.com?repo=repo&owner=owner',
+            allowAutoMerge: false,
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Set repository topics and enable GitHub Projects.',
+    example: yaml.stringify({
+      steps: [
+        {
+          action: 'github:repo:create',
+          name: 'Create a new GitHub repository with topics and GitHub Projects enabled',
+          input: {
+            repoUrl: 'github.com?repo=repo&owner=owner',
+            topics: ['opensource', 'nodejs', 'api'],
+            hasProjects: true,
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description:
+      'Create a private repository with collaborators having admin and write access.',
+    example: yaml.stringify({
+      steps: [
+        {
+          action: 'github:repo:create',
+          name: 'Create a new private GitHub repository with multiple collaborators',
+          input: {
+            repoUrl: 'github.com?repo=repo&owner=owner',
+            repoVisibility: 'private',
+            collaborators: [
+              { username: 'admin1', permission: 'admin' },
+              { username: 'writer1', permission: 'write' },
+            ],
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Disable branch deletion on merge.',
+    example: yaml.stringify({
+      steps: [
+        {
+          action: 'github:repo:create',
+          name: 'Create a new GitHub repository with branch deletion on merge disabled',
+          input: {
+            repoUrl: 'github.com?repo=repo&owner=owner',
+            deleteBranchOnMerge: false,
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description:
+      'Set repository visibility to internal and enable commit signing.',
+    example: yaml.stringify({
+      steps: [
+        {
+          action: 'github:repo:create',
+          name: 'Create a new internal GitHub repository with commit signing required',
+          input: {
+            repoUrl: 'github.com?repo=repo&owner=owner',
+            repoVisibility: 'internal',
+            requiredCommitSigning: true,
+          },
+        },
+      ],
+    }),
+  },
+  // {
```

Comment:
Can you please remove the commented code?

Reply:
Thank you for reviewing, requested changes are done. Please check